src/App.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import './App.css';
+import BooksList from './components/BooksList';
+import Catagories from './components/Categories';
+import Navigation from './components/Navigation';
+import AddBook from './components/AddBook';
+
+function App() {
+  return (
+    <div className="App">
+      <BrowserRouter>
+        <Navigation />
+        <Routes>
+          <Route exact path="/" element={<BooksList />} />
+          <Route exact path="/Catagories" element={<Catagories />} />
+          <Route exact path="/AddBook" element={<AddBook />} />
```

Comment:
- [ ] Here, I don't think you need `/AddBook` path since you are using this `AddBook` component under the `BooksList` component. Please remove this route from here and link from navbar as well.